op-service/flags/flags.go

```diff
@@ -0,0 +1,83 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+)
+
+const EnvVarPrefix = "SUPERCHAIN"
```

Comment:
Thoughts on name?

Reply:
Instead, you can expose a function that sets the correct env var prefix at runtime for shared flags like this.